### ISSUE TITLE
Percentage minor issue

### ISSUE THREAD

**Issue by djbox3** [autoNumeric/autoNumeric#443]
#### Steps to reproduce the problem

Using autoNumeric `2.0.12` and the browser Chrome or Firefox version `56.0.2924.87` on `Windows`,
**Setup:**
```html
<input type="text" data-a-sign=" %" data-p-sign="s">
```
```js
$('#demoPercentage').autoNumeric();
```

Enter in the following: `123.00-` (User shouldn't be able to enter a "-")

JS Error message : `"Uncaught Error: The value [-3.69] being set falls outside of the minimumValue [0] and maximumValue [9999999999999.9999] range set for this element"`

**Comment by AlexandreBonneau** [comment]
Hello @djbox3, would it be possible to create a codepen so we can reproduce the problem?

Thanks!

**Comment by BobKnothe** [comment]
@djbox3 

I have tried to replicate your issue in 2.xx and have unsuccessful - is it possible to post an example?

Bob

**Comment by djbox3** [comment]
https://jsfiddle.net/L14k8cxa

**Comment by djbox3** [comment]
https://jsfiddle.net/L14k8cxa
Sorry I wasn't able reproduce that specific java script error. But another issue is when you enter in "10.00- %". User shouldn't be able to enter in "-". 

When I paste in "10.00-" it would throw a Javascript error message "Uncaught Error: The pasted value '10.00- %' would result into an invalid content '10.00-'."

Also, I was able to enter in more values, but I already set the range 0 - 100. 
Example "1111111111110.00-%"
